idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/annotators/predef.kt

```diff
@@ -1,9 +1,49 @@
 package arrow.meta.ide.plugins.proofs.annotators
 
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.LocalQuickFixBase
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemDescriptorBase
+import com.intellij.codeInspection.ProblemHighlightType
 import com.intellij.openapi.editor.markup.EffectType
 import com.intellij.openapi.editor.markup.TextAttributes
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
 import java.awt.Color
 import java.awt.Font
 
 internal val implicitProofAnnotatorTextAttributes =
-  TextAttributes(null, null, Color(192, 192, 192), EffectType.WAVE_UNDERSCORE, Font.PLAIN)
\ No newline at end of file
+  TextAttributes(null, null, Color(192, 192, 192), EffectType.WAVE_UNDERSCORE, Font.PLAIN)
+
+fun addLocalQuickFix(
```

Comment:
check out the IdeUtil's those functions already exist. Same for `addProblemDescriptor`

Reply:
oh nice! didn't know that! thanks! 👍